### ISSUE TITLE
Streamed resolution doesn't update when server's resolution is reduced

### ISSUE THREAD

**Issue by coderthetyler** [pikvm/pikvm#821]
Been having some weird issues with the streamed resolution. 

**tl;dr: the active resolution can be increased, but once it's increased, it doesn't go back down again.**

I originally had the resolution on the server OS set to 1280x720 (16:9). No issues. I then bumped it up to 1440x900 (16:10). Again, no issues. I then dropped it back down to 1280x1080, and I got black bars on the top & bottom of the streamed video (presumably due to the now-mismatched aspect ratio). I ran a `v4lt2-ctl --all` and under DV Timings I saw the Active Width/Height reported as 1440x900. Uh oh.

I then tried updating the resolution on the server to 1920x1080. No issues. DV Timings now reports an active resolution of 1920x1080. Tried dropping the server resolution back to 1440x900 and, you guessed it, DV Timings still reports 1920x1080. I also saw black bars on the sides of the stream (again, presumably due to the mismatched aspect ratio of what's streamed vs what the server is outputting). Dropping the server resolution down to 1280x720 doesn't yield the black bars anymore but only because it's streaming at the same aspect ratio, but I do see scaling artifacts as if it's upscaling from 1280x720 to 1920x1080.

Things I tried without any luck:

- Rebooting my PiKVM
- Replugging the HDMI to the server
- Rebooting the server
- Various combinations & orderings of these things
- Played with the args to `ustreamer` (my most valiant attempt: removing `--dv-timings` and setting `--resolution 1366x768`, which just caused my stream to go blue, heh)
- Tried configuring the video device with `v4l2-ctl`

Here's what I'm working with: (I bought the pre-assembled PiKVM kit as I wasn't feeling too adventurous 😅.)
![image](https://user-images.githubusercontent.com/49089259/191615104-2e371286-5861-4ec9-985a-2cdd95dbc669.png)
![image](https://user-images.githubusercontent.com/49089259/191615136-a9c162a4-4c54-417c-b5d3-ef6ed30f0c15.png)

thanks :)

**Comment by coderthetyler** [comment]
Learn something new every day. Root cause of this issue is that:
- Increasing the display resolution increases the active signal resolution, and
- Decreasing the display resolution **does not** always decrease the active signal resolution.

You can see this mismatch in the Advanced Display Settings:
![image](https://user-images.githubusercontent.com/49089259/191626344-d75e6094-d3c6-4980-b385-d45aca361cdb.png)

The value for 'Active signal resolution' is what the PiKVM picks up. To change it, you just the display adapter options > list all modes > pick whatever resolution you want. This fixed the problem for me & the streamed resolution now matches what the desktop resolution is.

Thanks for the help!

**Comment by mdevaev** [comment]
:ok_hand:
